src/testIntegration/java/uk/gov/justice/digital/delius/jpa/standard/repository/OffenderRepositoryTest.java

```diff
@@ -1,31 +0,0 @@
-package uk.gov.justice.digital.delius.jpa.standard.repository;
```

Comment:
No idea what the point of this test is, I think it loaded an offender in the old UKCloud database